diagnostics/ENSO_RWS/shared/util.py

```diff
@@ -0,0 +1 @@
+
```

Comment:
remove empty file, or add contents back to file if it should be included

Reply:
the file is restored